include/lsst/afw/geom/Extent.h

```diff
@@ -306,6 +336,14 @@ Extent<T, 2>::Extent(Extent<U, 2> const &other) {
     this->setY(static_cast<T>(other.getY()));
 };
 
+// Should be consistent with converting constructor... don't know why
+// T = U instantiation is forbidden
```

Comment:
The point of my comment was not that a template constructor is not a copy constructor (I'm well aware of that), but that I can't delegate the copy-constructor to the pre-existing template and have to duplicate code instead.

I'm a little nervous about replacing non-trivial implementations, though, even if I think they "should" be equivalent. Do you think it does a lot of harm to leave the explicit code until somebody next works on the class?

Reply:
I don't think it does a lot of harm, but I do think it reduces a lot of the benefit of declaring these explicitly, and I think it might be worse than leaving them implicit.  IMO, the primary argument for declaring them explicitly is to tell any reader of the code, "we've thought about these operations; this is what the behavior should be".  If we're punting on actually thinking about what the behavior should be, what's the point of declaring them explicitly?